### DISCUSSION TITLE
Camera3D gizmo should reflect editor values

### DISCUSSION THREAD

**Discussion by marshalltower** [godotengine/godot-proposals#9739] [3D]
Just wanted to add some visibility to this item. Don't believe it's a bug, just a missed item that I notices after moving from unity.

When updating the the values on a Camera3D for properties "FOV","Near", and "Far", only "FOV" actually updates the gizmo. There's no plane shown whatsoever for "Near" and adjusting "Far" makes no changes to the gizmo length. This makes it really difficult when designing things in 3-dimensions to see what is actually in range of the camera; especially if the viewing plane isn't rectangular.

**Comment by timothyqiu** [comment]
Could add an `editor_draw_limits` option to `Camera3D` and make the gizmo draw a line representing the valid frustum range.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/8407.
